README.md

```diff
@@ -1,51 +1,238 @@
-4. [Latent Consistency Model (with LoRA) C++ image generation pipeline](./image_generation/lcm_dreamshaper_v7/cpp/README.md)
-
-### Requirements
-
-Requirements may vary for different samples. See respective readme files for more details,
-and make sure to install the OpenVINO version listed there. Refer to documentation to see
-[how to install OpenVINO](https://docs.openvino.ai/install).
-
-The supported devices are CPU and GPU including Intel discrete GPU.
-
-See also: https://docs.openvino.ai/2023.3/gen_ai_guide.html.
+OpenVINO™ GenAI is a library of most popular Generative AI model pipelines, optimized execution methods and samples that runs on top of highly performant [OpenVINO Runtime](https://github.com/openvinotoolkit/openvino).
+
+Library is friendly to PC and laptop execution, optimized for resource consumption and requires no external dependencies to run generative models and includes all required functionality (e.g. tokenization via openvino-tokenizers).
+
+![Text generation using LLaMa 3.2 model running on Intel ARC770 dGPU](./samples/generation.gif)
+
+## Supported Generative AI scenarios
+
+OpenVINO™ GenAI library provides very lightweight C++ and Python APIs to run following Generative Scenarios:
+ - Text generation using Large Language Models. For example, chat with local LLaMa model
+ - Image generation using Diffuser models, for example generation using Stable Diffusion models
+ - Speech recognition using Whisper family models
+ - Text generation using Large Visual Models, for instance Image analysis using LLaVa or miniCPM models family
+
+Library efficiently supports LoRA adapters for Text and Image generation scenarios:
+- Load multiple adapters per model
+- Select active adapters for every generation
+- Mix multiple adapters with coefficients via alpha blending
+
+All scenarios are run on top of OpenVINO Runtime that supports inference on CPU, GPU and NPU. See [here](https://docs.openvino.ai/2024/about-openvino/release-notes-openvino/system-requirements.html) for platform support matrix.
+
+## Supported Generative AI optimization methods
+
+OpenVINO™ GenAI library provides transparent way to use state of the art generation optimizations:
+- Speculative decoding that employs two models of different size and uses large model to periodically correct results of small model. See [here](https://pytorch.org/blog/hitchhikers-guide-speculative-decoding/) for more detailed overview
+- Lookahead decoding that attempts to guess multiple tokens based on historical context and corrects this guess using LLMs. See [here](https://lmsys.org/blog/2023-11-21-lookahead-decoding/) for more detailed overview
+- KVCache token eviction algorithm that reduces size of the KVCache by pruning less impacting tokens.
+
+Additionally, OpenVINO™ GenAI library implements continuous batching approach to use OpenVINO within LLM serving. Continuous batching library could be used in LLM serving frameworks and supports following features:
```

Comment:
should we also refer to OVMS docs as entry point to Continuous batching functionality?